flower/events.py

```diff
@@ -23,7 +25,13 @@
 
 class PrometheusMetrics(object):
     events = PrometheusCounter('flower_events_total', "Number of events", ['worker', 'type', 'task'])
-    runtime = Histogram('flower_task_runtime_seconds', "Task runtime", ['worker', 'task'])
+
+    runtime = Histogram(
+        'flower_task_runtime_seconds',
+        "Task runtime",
+        ['worker', 'task'],
+        buckets=options.task_runtime_metric_buckets
```

Comment:
@Tomasz-Kluczkowski env var FLOWER_TASK_RUNTIME_METRIC_BUCKETS not populating in options. something strange happening. not able to figure out. help me 
FYI I'm using development branch. the updated code is there in machine.

Reply:
@Tomasz-Kluczkowski can you pls look into this ?

Reply:
Hi,

I should have time later next week.
A bit busy with work/life currently but yes it's on my list, sorry for not showing a sign of life sooner...

Reply:
Hi,

I just had an initial look at this and know why we have this problem.

Unfortunately my broadband provider has an incident going on so I am just typing this on a phone.

We are using PrometheusMetrics class in EventState. The class has only class variables, meaning they are set in stone on initial import, waaay before the options object is modified by in command.py when the command line args are parsed.

We will have to refactor how this class is built so that we can pass config into it or delay setting instruments configurations.

Easiest first point of attack is to give it an __init__ method and set all the instruments in that method instead of making them class variables. Then they should be initialized correctly from that options. We could also just pass this runtime buckets setting into the init of PrometheusMetrics and decouple it from options...

@mher / @alienr ,let me know if you have other ideas. When I get some internet back I will try to push some proposed changes.